dto/src/main/scala/com/dwolla/cloudflare/domain/dto/ratelimits/RateLimitMatchDTO.scala

```diff
@@ -0,0 +1,26 @@
+package com.dwolla.cloudflare.domain.dto.ratelimits
+
+import com.dwolla.cloudflare.domain.dto.JsonWritable
+
+case class RateLimitMatchDTO (
+  request: RateLimitMatchRequestDTO,
+  response: Option[RateLimitMatchResponseDTO]
+) extends JsonWritable
+
+case class RateLimitMatchRequestDTO (
+  methods: Option[List[String]],
+  schemes: Option[List[String]],
+  url: String
+) extends JsonWritable
+
+case class RateLimitMatchResponseDTO (
+  status: Option[List[Int]],
```

Comment:
Could usages of `List[]` be converted to `Seq[]`? `List` imposes several implementation details that aren't necessarily required when a more abstract type would do.

Reply:
Good call. Updated places where List is not needed to Seq.